entgql/schema.go

```diff
@@ -265,6 +260,43 @@ func (e *schemaGenerator) buildTypes(g *gen.Graph, s *ast.Schema) error {
 	return nil
 }
 
+func (e *schemaGenerator) mayAddScalars(s *ast.Schema, def *ast.Definition) {
+	var redeclareErr bool
+	// If there is a config file but the schema there was not loaded.
+	if e.cfg != nil && e.cfg.Schema == nil {
+		// Do not fail in case of error.
+		err := e.cfg.LoadSchema()
```

Comment:
The schema may not in-complete to load, for example reference to our generated schema.